### DISCUSSION TITLE
Additional_data

### DISCUSSION THREAD

**Discussion by basakcelikdemir** [MAIF/shapash#577] [Q&A] [answered]
Hi,
When I add additional data, I see it on the right side of the modeled data in the Shapash monitor. However, I would like to see this data at the beginning of the data section instead.

How can I achieve this?

**Comment by guillaume-vignal** [comment, marked as answer]
Hi,

At the moment, Shapash does not support reordering additional data in the Shapash Monitor interface. By default, any additional data you add appears on the right side of the modeled data, and there is currently no built-in option to change this order.

However, I understand that having the ability to control the column order could improve the user experience. To address this, I have opened an issue on the Shapash repository to request this feature. You can follow the progress and contribute to the discussion here: [GitHub Issue Link](https://github.com/MAIF/shapash/issues/625)

Let me know if you have any other questions or suggestions!
